components/infobox/commons/infobox_building.lua

```diff
@@ -38,13 +38,19 @@ function Building:createInfobox()
 	local args = self.args
 
 	local widgets = {
-		Header{
-			name = self:nameDisplay(args),
-			image = args.image,
-			imageDefault = args.default,
-			imageDark = args.imagedark or args.imagedarkmode,
-			imageDefaultDark = args.defaultdark or args.defaultdarkmode,
-			size = args.imagesize,
+		Customizable{
+			id = 'header',
+			children = {
+				Header{
+					name = self:nameDisplay(args),
+					image = args.image,
+					imageDefault = args.default,
+					imageDark = args.imagedark or args.imagedarkmode,
+					imageDefaultDark = args.defaultdark or args.defaultdarkmode,
+					subHeader = self:subHeaderDisplay(args),
+					size = args.imagesize,
+				},
+			}
```

Comment:
you only overwrite the subheader display in the custom
do we need the whole header to be customizable?

Reply:
I took this from Infobox/unit though, so it's identical. But subheader would be enough, as you like.

Reply:
imo go with the smaller change (i.e. only add subheader support)